BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -454,7 +458,7 @@ struct TransactionConfirmationView: View {
       Label(Strings.Wallet.confirm, systemImage: "checkmark.circle.fill")
     }
     .buttonStyle(BraveFilledButtonStyle(size: .large))
-    .disabled(!confirmationStore.state.isBalanceSufficient)
+    .disabled(!confirmationStore.state.isBalanceSufficient || isConfirmDisabled)
```

Comment:
We also need to hide gas fee edit button & advanced settings row for Solana transactions IIRC.

Reply:
yeah i will update that in my next PR